src/main/resources/static/resources/subscription_templates.js

```diff
@@ -90,6 +90,7 @@ template_vars["templateEmailTrigger"] =
                     "created": "",
                     "notificationMeta" : "mymail@company.com",
                     "notificationType" : "MAIL",
+                    "emailSubject" : "",
```

Comment:
This needs to added in EI Backend subscription template file as well.

Reply:
If Anders havent already it done, I will do it in a separate commit.

Reply:
ok